source/includes/fact-minimum-versions.rst

```diff
@@ -2,6 +2,7 @@ Starting in version 1.2, ``mongosync`` can sync between clusters running
 some older (pre-6.0) releases of MongoDB Server. This feature requires
 additional preparation and configuration in pre-6.0 releases.
 
-If you would like to migrate pre-6.0 versions of MongoDB Server
-using ``mongosync``, please `contact <https://mongodb.com/contact>`__
-your account team to inquire about Professional Services.
+If you would like to migrate pre-6.0 versions of MongoDB Server using
+``mongosync``, or perform a cross version migration, please `contact
```

Comment:
```suggestion
If you would like to migrate from a pre-6.0 versions of MongoDB Server using
``mongosync`` please `contact
```

I think we can remove the `perform a cross version migration` since we just want users to contact professional service when the source is < 6.0 in the cross version case not all the cases. `migrate from a pre-6.0 versions of MongoDB Server` should cover all the cases including cross-version.

Reply:
Will do.